test/src/org/apache/jmeter/protocol/http/util/TestHTTPArgument.java

```diff
@@ -135,46 +135,44 @@ public void testUnsupportedEncodingException() throws Exception {
 
         @Test
         public void testWithShift_JIS_Encoding() throws Exception {
-        	
+            
```

Comment:
Is this change intentional?

Reply:
Created a new commit for these formatting related changes. Apologies for the unrelated change.